src/Ticket.php

```diff
@@ -3975,42 +3975,8 @@ public function showFormHelpdesk($ID, $ticket_template = false)
             $url_validate = Ticket::getSearchURL() . "?" . Toolbox::append_params($opt);
         }
 
-        $email  = UserEmail::getDefaultForUser($ID);
-        $default_use_notif = Entity::getUsedConfig('is_notif_enable_default', $_SESSION['glpiactive_entity'], '', 1);
-
        // Set default values...
-        $default_values = [
-            '_users_id_requester_notif' => [
-                'use_notification' => (($email == "") ? 0 : $default_use_notif)
-            ],
-            'nodelegate'                => 1,
-            '_users_id_requester'       => 0,
-            '_users_id_observer'        => 0,
-            '_users_id_observer_notif'  => [
-                'use_notification' => $default_use_notif
-            ],
-            'name'                      => '',
-            'content'                   => '',
-            'itilcategories_id'         => 0,
-            'locations_id'              => 0,
-            'urgency'                   => 3,
-            'items_id'                  => [],
-            'entities_id'               => $_SESSION['glpiactive_entity'],
-            'plan'                      => [],
-            '_add_validation'           => 0,
-            'type'                      => Entity::getUsedConfig(
-                'tickettype',
-                $_SESSION['glpiactive_entity'],
-                '',
-                Ticket::INCIDENT_TYPE
-            ),
-            '_right'                    => "id",
```

Comment:
This one is also absent from `Ticket::getDefaultValues()`, I do not know if it is used.